Makefile

```
@@ -5,3 +5,12 @@ include packaging/Makefile_packaging.mk
 
 clean:
 	rm -fr _topdir *.tar.gz
+
+sync_packaging:
+	rsync -aP --delete ../packaging/ ./packaging/ \
+		--exclude=.git \
+		--exclude=Makefile \
+		--exclude=README.md \
+		--exclude=packaging \
+		--exclude=Jenkinsfile \
+		--exclude=install
```

This is exactly what the `install` script in the _packaging_ repo does.  I'm not entirely against a `make` target (in `Makefile_packaging.mk`) for this in the _packaging_ repo if you wanted to submit a PR for that.  :-)

```suggestion
	../packaging/install
```

---

The install script seems to do some weird stuff with git hooks and fails:
```
+ rm -rf packaging/debian/ packaging/.git/ packaging/Jenkinsfile packaging/Makefile packaging/README.md packaging/_topdir packaging/v1.8.0.tar.gz packaging/install packaging/packaging.code-workspace
+ rm -f packaging/packaging
+ ln ../packaging/.git/hooks/applypatch-msg.sample ../packaging/.git/hooks/commit-msg.sample ../packaging/.git/hooks/fsmonitor-watchman.sample ../packaging/.git/hooks/post-update.sample ../packaging/.git/hooks/pre-applypatch.sample ../packaging/.git/hooks/pre-commit.sample ../packaging/.git/hooks/pre-merge-commit.sample ../packaging/.git/hooks/prepare-commit-msg.sample ../packaging/.git/hooks/pre-push.sample ../packaging/.git/hooks/pre-rebase.sample ../packaging/.git/hooks/pre-receive.sample ../packaging/.git/hooks/update.sample .git/hooks/
ln: failed to create hard link '.git/hooks/applypatch-msg.sample': File exists
ln: failed to create hard link '.git/hooks/commit-msg.sample': File exists
ln: failed to create hard link '.git/hooks/fsmonitor-watchman.sample': File exists
ln: failed to create hard link '.git/hooks/post-update.sample': File exists
ln: failed to create hard link '.git/hooks/pre-applypatch.sample': File exists
ln: failed to create hard link '.git/hooks/pre-commit.sample': File exists
ln: failed to create hard link '.git/hooks/pre-merge-commit.sample': File exists
ln: failed to create hard link '.git/hooks/prepare-commit-msg.sample': File exists
ln: failed to create hard link '.git/hooks/pre-push.sample': File exists
ln: failed to create hard link '.git/hooks/pre-rebase.sample': File exists
ln: failed to create hard link '.git/hooks/pre-receive.sample': File exists
ln: failed to create hard link '.git/hooks/update.sample': File exists
```
I guess those aren't fatal errors. Seems like it's assuming that my packaging repo has some git hooks set up?

In any case, I think I'm fine with the current approach, as I know what it does and how it works.